docs/casts/nipype_tutorial_intro.sh

```diff
@@ -0,0 +1,83 @@
+say "Import nipype building blocks"
```

Comment:
eh, someone really need to take care about #1778 and then this script should be contributed to nipype -- there is no reason for it to live hidden in datalad codebase